dora/core/server/worker/src/main/java/alluxio/worker/dora/PagedDoraWorker.java

```diff
@@ -961,9 +961,8 @@ public OpenFileHandle createFile(String path, CreateFilePOptions options)
       boolean overWrite = options.hasOverwrite() ? options.getOverwrite() : false;
       boolean exists = ufs.exists(path);
       if (!overWrite && exists) {
-        throw new RuntimeException(
-            new FileAlreadyExistsException(
-                String.format("File %s already exists but no overwrite flag", path)));
+        throw new AlreadyExistsException(String.format("File %s already exists"
+            + "but no overwrite flag", path));
```

Comment:
You should also provide reason literally for other reviewers.

Reply:
The RuntimeException is caught by DoraWorkClientServiceHandler, which is extracted by `AlluxioRuntimeException` like below.
```java
try {
    ...
    } catch (Exception e) {
      LOG.error(...);
      responseObserver.onError(AlluxioRuntimeException.from(e).toGrpcStatusRuntimeException());
    }
```
However, it is just a bare java.lang.RuntimeException, so it cannot be caught in [AlluxioRuntimeException.from()](https://github.com/Alluxio/alluxio/blob/2e1f95d5c9308a6037482b0b0ca7197801db2868/dora/core/common/src/main/java/alluxio/exception/runtime/AlluxioRuntimeException.java#L153), so finally the `FileAlreadyExistsException` becomes a `UnknownRuntimeException`, it is weird.

Reply:
If we use `AlreadyExistsException`, which is a child class of AlluxioStatusException, it will be handled in [here](https://github.com/Alluxio/alluxio/blob/2e1f95d5c9308a6037482b0b0ca7197801db2868/dora/core/common/src/main/java/alluxio/exception/runtime/AlluxioRuntimeException.java#L180C34-L180C34), So the problem is solved.